Resources/views/Form/formatter.html.twig

```diff
@@ -99,7 +99,9 @@
             {% elseif format == 'rawhtml' %}
                 elms.markItUp(markitup_sonataHtmlSettings);
             {% elseif format == 'richhtml' %}
-                {{ ckeditor_widget(form.vars.id, ckeditor_configuration) }};
+                {{ ckeditor_widget(form.vars.id, ckeditor_configuration, {
+                    input_sync: true,
```

Comment:
I just does the same thing as https://github.com/hug963/SonataFormatterBundle/blob/e7a0c994669914729b2d25945c9134751d3f6829/Resources/views/Form/formatter.html.twig#L23-L31
in the main widget, but with the built-in ckeditor way.
For now it is a bug as some forms will not update.
This change will not break the forms that already work.